src/analyze.ts

```diff
@@ -791,32 +816,36 @@ export default async function analyze(id: string, code: string, job: Job): Promi
     return true;
   }
 
+  function resolveAbsolutePathOrUrl (value: string | URL): string {
+    return value instanceof URL ? fileURLToPath(value as any) : value.startsWith('file:') ? fileURLToPath(new URL(value) as any) : path.resolve(value);
```

Comment:
You can remove the `as any` usages here if you do `import { URL } from 'url'` above, but not sure if this would cause other problems.